ui/app/components/namespace-picker.js

```diff
@@ -155,7 +155,9 @@ export default Component.extend({
 
   namespaceDisplay: computed('namespacePath', 'accessibleNamespaces', 'accessibleNamespaces.[]', function () {
     const namespace = this.namespacePath;
-    if (!namespace) return '';
+    if (!namespace) {
+      return 'root';
```

Comment:
It's best practice to delete the root user after setting up [vault](https://developer.hashicorp.com/vault/tutorials/operations/production-hardening#baseline-recommendations). Does that affect the namespace or is the default namespace always root? I believe it's the later, but thought it worth asking.